docs/File-Formats.md

```diff
@@ -1008,8 +1008,8 @@ CACO2<TAB>250<TAB>LAB_TEST<TAB>PSA<TAB>100
 This will lead to a timeline track for Lab_test with an additional subtrack specifically for PSA. PSA's events will be sized based on the result.
 
 
-## Gistic Data
-Running GISTIC 2.0 on e.g. GenePattern not only provides the [Discrete Copy Number Data](#discrete-copy-number-data), but also provides an amp_genes and a del_genes file. These cannot be directly imported into cBioPortal, but first have to be converted to a different file format. An example can be found in the [ACC TCGA study](https://github.com/cBioPortal/datahub/blob/master/public/acc_tcga/data_gistic_genes_amp.txt) on cBioPortal Datahub.
+## GISTIC 2.0 Data
+Running GISTIC 2.0 on e.g. (GenePattern)[https://cloud.genepattern.org] not only provides the [Discrete Copy Number Data](#discrete-copy-number-data), but also provides an amp_genes and a del_genes file. These cannot be directly imported into cBioPortal, but first have to be converted to a different file format. An example can be found in the [ACC TCGA study](https://github.com/cBioPortal/datahub/blob/master/public/acc_tcga/data_gistic_genes_amp.txt) on cBioPortal Datahub.
```

Comment:
@sandertan this link is wrongly made (should be `[xxx.com](xxx)`), otherwise it is not interpreted well.

Reply:
Good catch, fixed it.